DataFormats/Provenance/test/BuildFile.xml

```diff
@@ -8,6 +8,7 @@
   <use name="catch2"/>
 </bin>
 <bin   name="testProductResolverIndexHelper" file="testRunner.cpp productResolverIndexHelper_t.cppunit.cc">
+<use   name="cppunit"/>
```

Comment:
same

Reply:
is line 1 not meant to do that already? (eg difference of xml interpretation between scram and the python tool?)

Reply:
Yes but I never figured out how. I was modifying the origin which did not consider binaries to suit my needs.https://github.com/Teemperor/scram2cmake